infrastructures/infrastructure-common/src/main/java/org/ow2/proactive/resourcemanager/nodesource/infrastructure/ConnectorIaasController.java

```diff
@@ -76,6 +76,26 @@ public String createInfrastructure(String infrastructureId, String username, Str
         return infrastructureId;
     }
 
+    public String createInfrastructure(String infrastructureId, String username, String password, String endPoint,
+            String region, boolean destroyOnShutdown) {
+
+        String infrastructureJson = ConnectorIaasJSONTransformer.getInfrastructureJSONWithEndPoint(infrastructureId,
+                                                                                                   infrastructureType,
+                                                                                                   username,
+                                                                                                   password,
+                                                                                                   endPoint,
+                                                                                                   region,
+                                                                                                   destroyOnShutdown);
+
+        logger.info("Creating infrastructure : " + infrastructureJson);
```

Comment:
Is the infrastructureJson structure containing any sort of credentials ? if yes, it should not be put to logs.

Reply:
You make a great point. Since it's currently being printed in many places, I will do a clean up later in another PR.